controller/src/enhancements/player.rs

```diff
@@ -654,11 +676,19 @@ impl Enhancement for PlayerESP {
                     player_info.add_line(
                         esp_settings
                             .info_flags_color
-                            .calculate_color(player_rel_health),
+                            .calculate_color(player_rel_health, distance),
                         &player_flags.join(", "),
                     );
                 }
-                // TODO: Distance
+                if esp_settings.info_distance {
+                    let text = format!("[{:.0}]m", distance);
```

Comment:
Why do you display the distance in brackets?

Reply:
to me it looks cleaner this way, but if you prefer without ill take it out

Reply:
Done :)